#! /bin/sh /usr/share/dpatch/dpatch-run
## 04confdefs.dpatch by  <hesso@pool.math.tu-berlin.de>
##
## DP: Help configure by decreeing that some headers are present.

@DPATCH@
diff -Naur nvi-1.81.6.orig/build/confdefs.h nvi-1.81.6/build/confdefs.h
--- nvi-1.81.6.orig/build/confdefs.h	1970-01-01 01:00:00.000000000 +0100
+++ nvi-1.81.6/build/confdefs.h	2008-05-01 18:05:00.000000000 +0200
@@ -0,0 +1,3 @@
+
+#define HAVE_SYS_MMAN_H 1
+#define HAVE_SYS_SELECT_H 1
